3rdparty/qoi/readme.md

```diff
@@ -0,0 +1,33 @@
+qoi.h extemds for OpenCV
```

Comment:
typo

Reply:
Thank you for your comment, currently test runner is running.
So I will fix it after that(Ithink if I commit it, it will be canceled).

Reply:
Now I checked test runners, they are stopped. I'm sorry, I commit it.
(And I replace it to `Extended qoi.h for OpenCV`).